cos-fleetshard-operator-camel/src/main/java/org/bf2/cos/fleetshard/operator/camel/CamelOperandSupport.java

```diff
@@ -214,16 +177,8 @@ public static List<KameletEndpoint> createSteps(
             }
         }
 
-        for (JsonNode step : steps) {
-            var element = step.fields().next();
-
-            String templateId = shardMetadata.getKamelets().getProcessors().get(element.getKey());
-            if (templateId == null) {
-                throw new IllegalArgumentException("Unknown processor: " + element.getKey());
-            }
-
-            stepDefinitions.add(
-                configureStep(templateId, (ObjectNode) element.getValue()));
+        if (!connectorConfiguration.getProcessorsSpec().isEmpty()) {
```

Comment:
can we add a flag to toggle processors ?

Reply:
Done